.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Description
+Please include a summary of change(s) and/or link to associated [issue](https://github.com/SAP/open-ux-tools/issues)
+
+## Type of change
+- [ ] Bug (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds a new feature)
+- [ ] Breaking change (Bug or New feature that would cause existing functionality/consumers to not work as expected)
+- [ ] Non-Breaking chores (Changes to tools, libraries, build process, documentation, etc)
+- [ ] None of the above (Reviewers might ask for more clarification)
+
+## How have you tested?
+Make it easier for reviewer by describing the tests you have done to verify your changes and provide clear instructions for reviewers to reproduce.
+
+## Checklist:
```

Comment:
Checklist before asking for review/checklist before merging? I'm asking because it contains the item `...test pass locally`. For a PR to be merged in the tests have to pass on CI.

Reply:
hello @ullasholla, would you have better suggestion? goal is to get a good faith commitment from the pr author that they have made an attempt to ensure tests passes locally, not like using `--no-verify`. Thoughts?

Reply:
> goal is to get a good faith commitment from the pr author that they have made an attempt to ensure tests passes locally

In that case, this may not be required? PRs can't be merged in without tests passing and exceeding the coverage threshold. From my point of view, it's the PR author's responsibility to fix failing tests on CI, if any.

I see your point though of making them aware. I'm good either keeping it as it is or removing it entirely. Feel free to resolve the conversation

Reply:
ok point taken. Lets keep it for now and lets see some future PRs and we can adapt accordingly. thanks much